sqlalchemy_continuum/manager.py

```diff
@@ -24,7 +25,7 @@ def wrapper(self, mapper, connection, target):
         try:
             uow = self.units_of_work[conn]
         except KeyError:
-            uow = self.units_of_work[conn.engine]
+            uow = self.units_of_work.get(conn.engine, None)
```

Comment:
Worth a `logger.info` here?

Reply:
Yea I'll put something in here - I wonder if I should be even more strict, because when native mode is off, a uow should always be found by this function.  my gut is actually to raise iff native is off.

Will definitely take a look at this, thanks for flagging!